tests/test_post.py

```diff
@@ -112,312 +112,6 @@ def plastic_solve(mapdl):
     mapdl.mute = False
 
 
-@pytest.fixture(scope="module")
-def contact_solve(mapdl):
-    mapdl.mute = True
-    mapdl.finish()
```

Comment:
why is this test removed?

Reply:
This test was moved to ``confest`` because it is going to be used in more places than the post module.